src/deb.cc

```diff
@@ -27,3 +27,16 @@ Json::Value DebianManager::getInstalledPackages() {
   dpkg_program_done();
   return packages;
 }
+
+data::InstallOutcome DebianManager::install(const Uptane::Target &target) const {
+  std::string cmd = "dpkg -i ";
```

Comment:
Does this work in the case where Aktualizr is installing an update to Aktualizr?